drivers/at86rf2xx/at86rf2xx.c

```diff
@@ -100,14 +113,9 @@ void at86rf2xx_reset(at86rf2xx_t *dev)
         at86rf2xx_set_state(dev, AT86RF2XX_STATE_FORCE_TRX_OFF);
     }
 
-    /* get an 8-byte unique ID to use as hardware address */
-    luid_get(addr_long.uint8, IEEE802154_LONG_ADDRESS_LEN);
-    /* make sure we mark the address as non-multicast and not globally unique */
-    addr_long.uint8[0] &= ~(0x01);
-    addr_long.uint8[0] |=  (0x02);
     /* set short and long address */
-    at86rf2xx_set_addr_long(dev, ntohll(addr_long.uint64.u64));
-    at86rf2xx_set_addr_short(dev, ntohs(addr_long.uint16[0].u16));
+    at86rf2xx_set_addr_long(dev, dev->netdev.long_addr);
+    at86rf2xx_set_addr_short(dev, dev->netdev.short_addr);
```

Comment:
Is it wise, just to assume the system is little-endian?

Reply:
I would say so. RIOT does not support any big endian systems and that assumption is made in most places where RIOT talks to devices with little-endian multi-byte registers.

AFAIR there are no big endian MCUs.

Reply:
A byte array has no byte order. It makes no sense to convert byte order of a byte array.

Reply:
True dat :sweat_smile: